workflow-testing/src/jvmMain/kotlin/com/squareup/workflow1/testing/WorkerWorkflowHelper.kt

```diff
@@ -15,4 +16,4 @@ import kotlin.reflect.KType
  * See https://youtrack.jetbrains.com/issue/KT-20760.
  */
 internal fun Workflow<*, *, *>.workerWorkflowWorkerTypeOrNull(): KType? =
-  (this as? com.squareup.workflow1.WorkerWorkflow)?.workerType
+  (this as? HasWorkerType)?.workerType
```

Comment:
Need to update the kdoc above to explain how this is done now through the interface.